hack/lib/build/images.sh

```diff
@@ -64,15 +65,21 @@ readonly -f os::build::image
 function os::build::image::internal::generic() {
 	local directory=$2
 
+	local result='1'
 	if os::util::find::system_binary 'imagebuilder' >/dev/null; then
-		os::build::image::internal::imagebuilder "$@"
```

Comment:
This is a classic case of `set -o errexit` being on but being ignored inside of a function call because the function call is inside of an `if` block.